Text_to_Translated_Text_Step_2.ipynb

```diff
@@ -0,0 +1,360 @@
+      },
+      "execution_count": 130,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Requirement already satisfied: mtranslate in /usr/local/lib/python3.10/dist-packages (1.8)\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Import the 'mtranslate' library\n",
+        "from mtranslate import translate\n",
+        "\n",
+        "start_time = time.time()\n",
+        "\n",
+        "def translate_to_spanish(text):\n",
+        "    try:\n",
+        "        translated_text = translate(text, 'es', 'en')\n",
+        "        return translated_text\n",
+        "    except Exception as e:\n",
+        "        return f\"Translation error: {str(e)}\"\n",
+        "\n",
+        "if __name__ == \"__main__\":\n",
+        "    # Define the text to be translated\n",
+        "    text_to_translate = \" Hello there! My name is Phil. I have been an online English tutor for almost three years now. I've been helping students from beginner to advanced level improve their English skills. I've tried teaching kids and adults. I used CPR. Look! Rewards! And many more! I would love to help you improve and make progress in learning. From phonics, grammar, pronunciation and up to conversational English. I've already earned my Tassel and Tia Felt certificate, so you're in good hands. Learn with me and let's make English a fun and easy language. See ya!\"\n",
+        "    translated_text = translate_to_spanish(text_to_translate)\n",
+        "    print(translated_text)\n",
+        "\n",
+        "\n",
+        "print(\"MyMemory : %s seconds\" % (time.time() - start_time))"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "Ft-xGTgjtd1u",
+        "outputId": "7ae5a031-2a0d-4700-fa5f-9b3d92c2cd90"
+      },
+      "execution_count": 131,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "¡Hola! Mi nombre es Fil. He sido tutor de inglés en línea durante casi tres años. He estado ayudando a estudiantes desde nivel principiante hasta avanzado a mejorar sus habilidades en inglés. He intentado enseñar a niños y adultos. Utilicé RCP. ¡Mirar! ¡Recompensas! ¡Y muchos más! Me encantaría ayudarte a mejorar y progresar en el aprendizaje. Desde fonética, gramática, pronunciación y hasta inglés conversacional. Ya obtuve mi certificado de Borla y Tia Felt, así que estás en buenas manos. Aprende conmigo y hagamos del inglés un idioma fácil y divertido. ¡Nos vemos!\n",
+            "MyMemory : 0.4078376293182373 seconds\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Time Taken to Translate by Different models\n",
+        "\n",
+        "| Model    | Time of Execution (in sec)|\n",
```

Comment:
Besides the speed, what is the accuracy of the translation?

Reply:
There is no direct way to check the accuracy. so i have changed the translation language from Spanish to Hindi, a language which we are familiar with. So that we can manually check the accuracy of the translated text. We will try to find a better way of checking the accuracy but mostly these models are very accurate.